patches/chrome-browser-ui-libgtkui-gtk_util.cc.patch

```diff
@@ -1,13 +0,0 @@
-diff --git a/chrome/browser/ui/libgtkui/gtk_util.cc b/chrome/browser/ui/libgtkui/gtk_util.cc
-index dc3cc6806b286d37f1002e41b3dec2fccf85d0ff..f5d3cd2f3dc5faa9a43423454731cbd4616e78d3 100644
---- a/chrome/browser/ui/libgtkui/gtk_util.cc
-+++ b/chrome/browser/ui/libgtkui/gtk_util.cc
-@@ -98,7 +98,7 @@ std::string GetDesktopName(base::Environment* env) {
-   std::string name;
-   if (env->GetVar("CHROME_DESKTOP", &name) && !name.empty())
-     return name;
--  return "chromium-browser.desktop";
-+  return "brave-browser.desktop";
```

Comment:
?

Reply:
GetDesktopName was removed from here and moved to channel_info.h 

https://chromium.googlesource.com/chromium/src/+/b8a8603ff3b2caf0106edec3d33093125e4a67cf